pkg/xen-tools/Dockerfile

```diff
@@ -60,15 +62,22 @@ WORKDIR /
 
 # Download and verify xen
 #TODO: verify Xen
-RUN \
-    [ -f "$(basename ${XEN_SOURCE})" ] || curl -fsSLO "${XEN_SOURCE}" && \
-    tar --absolute-names -xz < "$(basename ${XEN_SOURCE})" && mv "/xen-${XEN_VERSION}" /xen
+# hadolint ignore=DL3020
+ADD ${XEN_SOURCE} /xen.tar.gz
+RUN [ -f "$(basename ${XEN_SOURCE})" ] || tar --absolute-names -xz < /xen.tar.gz && mv "/xen-${XEN_VERSION}" /xen
 
 # Apply local patches
 COPY patches-${XEN_VERSION} /patches
 WORKDIR /xen
 RUN cat /patches/*.patch /patches/"$(uname -m)"/*.patch | patch -p1
 
+# we will need this to build xen; without it, it will try and download directly
+ENV SEABIOS_UPSTREAM_URL /seabios.git
```

Comment:
It actually is. The `./configure` uses that URL to determine where to get seabios.

Reply:
Ah, interesting, you defined it for internal logic of the builder. Good to know, thank you. Probably it would be nice to mention that we should not rename it.

Reply:
That is a very good idea. Doing it now.

Reply:
Done